src/config/Settings.ts

```diff
@@ -0,0 +1,28 @@
+import credentialsData from 'src/credentials.json'
```

Comment:
This is duplicated in at least 2 more RPs.

Reply:
you're right